src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewPropertiesOutput.java

```diff
@@ -329,7 +329,7 @@ protected void doUpdate(ModelPart part) {
             if (context.config.containsCriterion(EDDifferentialPrivacy.class)) {
                 EDDifferentialPrivacy criterion = context.config.getCriterion(EDDifferentialPrivacy.class);
                 Property n = new Property(Resources.getMessage("PropertiesView.51"), new String[] { Resources.getMessage("PropertiesView.141") }); //$NON-NLS-1$ //$NON-NLS-2$
-                new Property(n, Resources.getMessage("PropertiesView.142"), new String[] { SWTUtil.getPrettyString(criterion.getEpsilon())}); //$NON-NLS-1$
+                new Property(n, Resources.getMessage("PropertiesView.142"), new String[] { SWTUtil.getPrettyString(criterion.getEpsilonAnon()+criterion.getEpsilonSearch())}); //$NON-NLS-1$

```

Comment:
Data-dependent DP should be implemented *in addition* to the already existing implementation of data-independent DP. Is this considered here accordingly? Also: in case of data-dependent DP Eanon and Esearch could be listed separately (e.g. in parantheses following the sum).